packages/flutter/lib/src/foundation/shader_warm_up.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:ui';
```

Comment:
please see the README in this directory.

This probably belongs in the `painting` library, not `foundation`.

Reply:
Moved to painting. Painting was my first choice but analyze won't allow me to do that since I used it in scheduler, which creates a dependency loop. Now I've removed all changes to scheduler and `ShaderWarmUp` is in painting again.